lib/rules/no-else-return.js

```diff
@@ -33,7 +35,45 @@ module.exports = {
          * @returns {void}
          */
         function displayReport(node) {
-            context.report({ node, message: "Unnecessary 'else' after 'return'." });
+            context.report({
+                node,
+                message: "Unnecessary 'else' after 'return'.",
+                fix: fixer => {
+                    const sourceCode = context.getSourceCode();
+                    const startToken = sourceCode.getTokenByRangeStart(node.start);
+                    const elseToken = sourceCode.getTokenBefore(startToken);
+                    const source = sourceCode.getText(node);
+                    const lastIfToken = sourceCode.getTokenBefore(elseToken);
+                    let fixedSource;
+
+                    if (startToken.type === "Punctuator" && startToken.value === "{") {
+                        const firstTokenOfElseBlock = sourceCode.getTokenAfter(startToken);
+
+                        if (lastIfToken.value !== "}" &&
```

Comment:
Hmm makes sense, but `node.parent` is not defined here - could you recommend a better way to do that check?